common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.inventory;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.mc.event.ContainerClickEvent;
+import com.wynntils.mc.event.ContainerCloseEvent;
+import com.wynntils.mc.event.ScreenInitEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@ConfigCategory(Category.INVENTORY)
+public class CustomBankPagesFeature extends Feature {
+    // If possible, lock these to only allow between 1-21
```

Comment:
You could use the onConfigUpdate event to check if any of these are invalid, then reset it and tell the user to fix it.

Reply:
Or we should extend our config types to include a ranged integer...

Reply:
I'll use onConfigUpdate for now, but if/when ranged integer gets added it can be changed to that